bin/_tag.sh

```diff
@@ -2,8 +2,8 @@
 
 set -eu
 
-git_sha() {
-    git rev-parse "$1" | cut -c 1-8
+git_sha_head() {
+    git rev-parse HEAD | cut -c 1-8
```

Comment:
is this equivalent to `git rev-parse --short=8 HEAD`?

Reply:
Whoa, I had no idea `--short` took a parameter, and the default is 7. Will update to this.